BraveWallet/Crypto/Onboarding/PasswordValidation.swift

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+struct PasswordValidation {
+  /// Whether or not a password is valid to use when creating or restoring a wallet
+  ///
+  /// Currently a strong password is defined by having at least 7 characters, and including at least
+  /// 1 number and 1 special character.
+  ///
+  /// The available special characters are:
+  ///
+  ///      \` ~ ! @ # $ % ^ & * ( ) + = ? ; : | < > , . ' " { } / [ ] \ - _
+  static func isValid(_ password: String) -> Bool {
```

Comment:
Hehe yes there were tested for this :P

Reply:
cc @SergeyZhukovsky ; this isn't the same as the Android requirement: https://github.com/brave/brave-core/blob/e4bceafcb3999b4b55818dacb269338bb144e21b/android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java#L99

it would be best if we could consolidate all these password checks into a single function across all platforms.